frontend/src/lib/components/LemonTable/LemonTable.tsx

```diff
@@ -20,6 +20,7 @@ function determineColumnKey(column: LemonTableColumn<any, any>, obligationReason
 function determineColumnKey(column: LemonTableColumn<any, any>, obligationReason?: string): string | null {
     const columnKey = column.key || column.dataIndex
     if (obligationReason && columnKey == null) {
+        // == is intentional to catch undefined too
```

Comment:
```suggestion
    if (obligationReason && (columnKey === null || columnKey === undefined)) {
```

Reply:
Fair enough though in this case the extra parentheses make things a bit harder to read… I'll merge as is for now and keep this in mind